types/typegen/generator.go

```diff
@@ -80,7 +80,7 @@ func (g *Generator) Find() error {
 		return err
 	}
 	g.Types = []*Type{}
-	err = generate.Inspect(g.Pkg, g.Inspect)
+	err = generate.Inspect(g.Pkg, g.Inspect, "enumgen.go", "typegen.go")
```

Comment:
Is typegen necessarily exclusive of enumgen.go files? It would also fix the other comment in enumgen above to remove that.

Reply:
yes definitely: no way you would want to add either to either.